src/test/java/uk/gov/hmcts/reform/em/annotation/rest/errors/ExceptionTranslatorIntTest.java

```diff
@@ -13,14 +13,17 @@
 
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
 /**
  * Test class for the ExceptionTranslator controller advice.
  *
  * @see ExceptionTranslator
  */
 @RunWith(SpringRunner.class)
+@SuppressWarnings({"AbbreviationAsWordInName"})
```

Comment:
same here, we can try to fix it instead of suppressing.

Reply:
okay